infra/src/dsl/xroad.ts

```diff
@@ -840,6 +840,17 @@ export const UniversityGatewayReykjavikUniversity = new XroadConf({
   },
 })
 
+export const JudicialSystemServicePortal = new XroadConf({
+  env: {
+    XROAD_JUDICIAL_SYSTEM_SP_PATH: {
+      dev: 'IS-DEV/GOV/10014/Rettarvorslugatt-Private/judicial-system-mailbox-api',
+      staging:
```

Comment:
The staging url should pint to `IS-TEST/GOV/etc...`

The prod organization id probably isn't correct, shouldn't be identical to the dev one.